Library/Formula/ansible.rb

```diff
@@ -22,6 +22,7 @@ class Ansible < Formula
   depends_on :python if MacOS.version <= :snow_leopard
   depends_on "libyaml"
   depends_on "openssl"
+  depends_on "libffi" if OS.linux?
```

Comment:
Also please report the error message that you see without this patch. `brew gist-logs ansible` is helpful.